First inclusion of DavResource

Hofer-Julian opened the pull request:
Note: does not build yet

Commit:
First inclusion of DavResource with isahc

bilelmoussaoui commented on Cargo.toml:
```diff
@@ -5,3 +5,7 @@ authors = ["Bilal Elmoussaoui <bil.elmoussaoui@gmail.com>", "Julian Hofer"]
+isahc = "0.9"
+quick-xml = "0.18"
+thiserror = "1.0"
+mockito = "0.25.2"
```
I tend to avoid using enforcing micro releases ^^

Hofer-Julian replied:
Actually me as well, that wasn't done on purpose :)

Hofer-Julian replied:
Fixed

Commit:
Enable basic tests

Commit:
Modify tests

- remove dummy test from lib.rs
- change test so that it tests an actual additional verb from webdav
